mentorship iosTests/SettingsTests.swift

```diff
@@ -127,4 +127,25 @@ class SettingsTests: XCTestCase {
         
         wait(for: [expectation], timeout: 1)
     }
+    
+    func testChangePasswordReset() throws {
+        
+        // View Model
```

Comment:
1. Do we need this to be a throwing function?
2. Can we remove this additional line break for consistency and neatness...

Lastly, just do a final self review.